pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2452,12 +2491,20 @@ func (c *Cloud) EnsureLoadBalancer(clusterName string, apiService *api.Service,
 		return nil, err
 	}
 
+	// Add loadbalancer's security group's rules into instances' security groups' rules.
```

Comment:
How about "Add rules to the instances' security groups to allow traffic from the load balancer security groups"